src/backend/distributed/operations/worker_split_shard_replication_setup_udf.c

```diff
@@ -53,6 +53,12 @@ static void PopulateShardSplitInfoInSM(ShardSplitInfoSMHeader *shardSplitInfoSMH
 static void ReturnReplicationSlotInfo(Tuplestorestate *tupleStore,
 									  TupleDesc tupleDescriptor);
 
+typedef struct GroupedShardSplitInfos
```

Comment:
could use some comments on what this represents

Reply:
done